### DISCUSSION TITLE
Embedding video in revealjs

### DISCUSSION THREAD

**Discussion by ashleyleech** [quarto-dev/quarto-cli#3781] [Q&A] [answered]
When I try to embed a video, such as: 

{{< video https://www.youtube.com/watch?v=9X68dm92HVI width="100%" height="100%" >}}

I get the following error: 

No video source specified for video shortcode

Any suggestions on what I can do to ensure that the video is embedded?

**Comment by mcanouil** [comment, marked as answer]
I believe you are not using the proper URL for videos, see <https://quarto.org/docs/authoring/videos.html>.

Edit: contrary to what is written the `watch?v=` URLs are not valid.

**Reply by ashleyleech** [reply]
Thank you! I replaced the "watch?v=" with "embed" & it worked. Thanks again :)

**Reply by cscheid** [reply]
That `watch?v=` syntax will likely work on `main` now.
